gr-utils/python/blocktool/cli/parseheader.py

```diff
@@ -0,0 +1,299 @@
+#
+# Copyright 2019 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+""" Module to generate parsed header output data """
+
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+
+import os
+import json
+import logging
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except:
+    from yaml import Loader, Dumper
+
+import click
+
+from ..cli.base import BlockToolException, run_blocktool
+from ..core.parseheader import BlockHeaderParser
+from ..core import Constants
+
+LOGGER = logging.getLogger(__name__)
+
+## setup dumper for dumping OrderedDict ##
+_MAPPING_TAG = yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG
+
+
+def dict_representer(dumper, data):
+    """ Representer to represent special OrderedDict """
```

Comment:
"Representer to represent..." seems a bit redundant

Reply:
Representing with Ordered dict and then using it to convert in YAML parameters is quite easy. The whole blocktool yaml_generator is based on this.

Reply:
I'm not referring to the implementation or return, but to the grammar of the docstring :)